website_sale_product_brand/README.rst

```diff
@@ -0,0 +1,95 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================
+Website Sale Product Brand
+==========================
+
+Description
+===========
+
+This module was written to extend the functionality of product filtering on website.
+It will allow you to filter product based on its brand.
+
+While shopping online, we have seen various eShops having a feature to shop by brands
+which ODOO does not yet provide officially.Website Sale Product Brand fills the gap at certain
+extent and by providing basic search by brands, thus reducing end-user’s efforts in 
+searching the products he/she wants to purchase.
+
+Installation
+============
+
+To install this module, you need to install following module:
+->https://github.com/OCA/product-attribute/tree/11.0/product_brand
+
+Usage
+=====
+Shop by brand feature is available on various famous e-commerce websites like amazon, flipkart and many.
+Shop by brand feature enables you to display product relevant to that particular brand.
+
+To use this module, you need to:
+
+Once you install this module, user will be able to create a new brand and define the brand to a product.
+
+- To create product brand
+    go to:
+        sales>products>product brands
+
+User can assign a nice logo with brand description.
+
+- After configuring the brand, user can assign a particular brand to a particular products.
+
+Based on this configuration, you will see the menuitem shop by brand next to shop menu.
+It will show all the brands and once you select that brand it will show product's which
+is related to this brand.
+
+The blog here explains the HOWTO :
+    http://www.serpentcs.com/serpentcs-odoo-ecommerce-shop-brands-contribution
```

Comment:
Do this link give any relevant info? The README.rst info should be the more self contained possible

Reply:
Indeed no links outside OCA